consensus/ethash/ethash.go

```diff
@@ -459,6 +462,9 @@ type Ethash struct {
 	lock      sync.Mutex      // Ensures thread safety for the in-memory caches and mining fields
 	closeOnce sync.Once       // Ensures exit channel will not be closed twice.
 	exitCh    chan chan error // Notification channel to exiting backend threads
+
+	// Test relative attributes
+	disableRemoteVerify bool
```

Comment:
I think @ppratscher asked for this as a feature to permit disabling ethash verification altogether (since the pool verifies it already). We could convert this into an actual feature with `--miner.noverify`.